src/api/modes.ts

```diff
@@ -62,3 +61,22 @@ export async function toMode(modeName: string, count?: number) {
       }));
   }, 0);
 }
+
+/**
+ * Returns the {@link Mode} with the given name, prioritizing one that's within the name "namespace"
```

Comment:
optional nit: my bad, my suggestion had a typo
```suggestion
 * Returns the {@link Mode} with the given name, prioritizing one that's within the same "namespace"
```